torch/csrc/jit/python/python_sugared_value.cpp

```diff
@@ -217,6 +217,32 @@ std::shared_ptr<SugaredValue> PythonModuleValue::attr(
   return toSugaredValue(member, m, loc, /*is_constant=*/true);
 }
 
+std::shared_ptr<SugaredValue> CUDAPythonModuleValue::attr(
+    const SourceRange& loc,
+    Function& m,
+    const std::string& field) {
+
+  if(field == "_cuda_getDevice") {
+    return std::make_shared<BuiltinFunction>(Symbol::aten("get_device"), c10::nullopt);
+  } else if(field == "_cuda_getDeviceIndex") {
+    return std::make_shared<BuiltinFunction>(Symbol::aten("get_device_index"), c10::nullopt);
+  } else if(field == "_cuda_getDeviceCount") {
+    return std::make_shared<BuiltinFunction>(Symbol::aten("get_device_count"), c10::nullopt);
+  } else if(field == "_cuda_setDevice") {
+    return std::make_shared<BuiltinFunction>(Symbol::aten("set_device"), c10::nullopt);
+  } else if(field == "_cuda_setStream") {
+    return std::make_shared<BuiltinFunction>(Symbol::aten("setCudaStream"), c10::nullopt);
+  } else if(field == "_cuda_getStream") {
+    return std::make_shared<BuiltinFunction>(Symbol::aten("getCudaStream"), c10::nullopt);
```

Comment:
This code can be made less repetitive by refactoring the pairs of function names and op names into a map. Ideally we would use the same names for both so that even a map is not needed.